<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find Rack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1921</width>
      <height>1051</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="options_lbl">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="response_lbl">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>330</y>
      <width>671</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="search_lbl">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>990</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>arrow.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="user_prompt_txtedit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>661</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>190</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Qwen-Qwq-32b</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>210</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Deepseek-R1-Distill-Llama-70b</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>230</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Gemma2-9b-It</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>250</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Compund-Beta-Mini</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>270</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Compound-Beta</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>290</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Llama-3.3-70b-Versatile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>390</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Meta-Llama/Llama-4-Maverick-17b-128e-insturct</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>370</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Meta-Llama/Llama-4-Scout-17b-16e-instruct</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>310</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Llama-3.1-8b-instant</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>330</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Llama3-70b-8192</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1680</x>
      <y>350</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 2px solid transparent;
}

QLabel:hover {
    border: 2px solid #844b36
}</string>
    </property>
    <property name="text">
     <string>Llama3-8b-8192</string>
    </property>
   </widget>
   <widget class="QLabel" name="current_model_lbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>980</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tool_used_lbl">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>950</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>950</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ðŸŸ¢ Agent Used :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
